app/src/main/java/cto/its/ny/gov/fakenews/util/services/NewsService.kt

```diff
@@ -0,0 +1,32 @@
+package cto.its.ny.gov.fakenews.util.services
+
+import cto.its.ny.gov.fakenews.model.Article
+import cto.its.ny.gov.fakenews.model.Source
+import cto.its.ny.gov.fakenews.util.constants.SourceCategories
+import cto.its.ny.gov.fakenews.util.constants.SourceLanguages
+
+class NewsService {
+
+    fun createArticles(amountOfArticles: IntRange): ArrayList<Article> {
+
+        val arrayOfAticles = arrayListOf<Article>()
+        for (index in amountOfArticles) {
+            val src = Source(name = "Newspaper" + index, category = SourceCategories.SCIENCE, language = SourceLanguages.ar)
```

Comment:
nit: you could do string interpolation here with the `name` param:

```suggestion
            val src = Source(name = "Newspaper $index", category = SourceCategories.SCIENCE, language = SourceLanguages.ar)
```